script.module.t1mlib/addon.xml

```diff
@@ -13,7 +13,7 @@
         <email></email>
         <website></website>
         <forum>http://forum.kodi.tv/showthread.php?tid=247163</forum>
-        <source>https://github.com/learningit/repository.t1m.v2.addons/tree/master/src/script.module.t1mlib</source>
+        <source>https://github.com/xbmc/repo-scripts/tree/isengard/script.module.t1mlib</source>
```

Comment:
Why?
On May 6, 2016 11:02 AM, "Razzeee" <notifications@github.com> wrote:

> In script.module.t1mlib/addon.xml
> <https://github.com/xbmc/repo-scripts/pull/146#discussion_r62341903>:
>
> > @@ -13,7 +13,7 @@
> >          <email></email>
> >          <website></website>
> >          <forum>http://forum.kodi.tv/showthread.php?tid=247163</forum>
> > -        <source>https://github.com/learningit/repository.t1m.v2.addons/tree/master/src/script.module.t1mlib</source>
> > +        <source>https://github.com/xbmc/repo-scripts/tree/isengard/script.module.t1mlib</source>
>
> This should be your own repo not this one.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/repo-scripts/pull/146/files/e48fbb6a06753dfaa2256cfc2e894d7a276b8d02#r62341903>
>